modules/aws-api-gw/api_gateway.tf

```diff
@@ -0,0 +1,28 @@
+resource "aws_api_gateway_rest_api" "api_gw" {
+  #file("${path.module}/sample.json")
```

Comment:
please 
  rename module to api-gateway,
  have this file renamed to main.tf
  remove the above commented file
  use terraforms template instead if sample.json file and pass configurable params to template
  have some minimal info in outputs.tf

Reply:
Everything is okay, but I cant use terraform template for jsonencode function, at this time "body" is just hardcoded in jsonencode block